runtime/tools/spec-builder/test/build-spec.spec.js

```diff
@@ -0,0 +1,37 @@
+const assert = require('assert');
+const fs = require('fs');
+const { buildSpec, emptyActualRawFiles } = require('./utils/spawn-child-process');
+const { isEqual } = require('lodash');
```

Comment:
You do not need to use `lodash` for the `isEqual` function. You can see the `nodejs` `assert.deepEqual()`. That's one dependency less, so yeah.